### PR TITLE
chore: react-skeleton initial implementation

### PR DESCRIPTION
**Pull request by tomi-msft**
This PR is for the initial implementation of `Skeleton`. It adds initial props, styles, and tests. These may change later.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/a180b67ce1107e8d4cb02aceb2e92a9e162b532e'>a180b67ce1107e8d4cb02aceb2e92a9e162b532e</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by tomi-msft** [comment]
Fixes #25448

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/Avatar-mount_base.html">1254</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/Avatar-mount.html">1290</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/Button-mount_base.html">912</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/Button-mount.html">916</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/FluentProvider-mount_base.html">1503</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/FluentProvider-mount.html">1494</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/FluentProviderWithTheme-mount_base.html">579</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/FluentProviderWithTheme-mount.html">586</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">541</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">551</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">577</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">582</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/InfoButton-mount_base.html">527</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/InfoButton-mount.html">522</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/MakeStyles-mount_base.html">1977</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/MakeStyles-mount.html">1960</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/Persona-mount_base.html">2827</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/Persona-mount.html">2804</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/SpinButton-mount_base.html">2353</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test-react-components/SpinButton-mount.html">2361</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: README.md [hunk 1]
```diff
@@ -35,8 +35,8 @@ The following table will help you navigate the 3 projects and understand their d
 | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
```
Comment (fabricteam) [inline README.md:35]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=25452&commitId=019bd59c7b4a752fa6658646e4761a86f7983948&env=prod)

✅ There was 4 screenshots added, 0 screenshots removed, 1041 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/BaseButton-mount_base.html">1657</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/BaseButton-mount.html">1614</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Breadcrumb-mount_base.html">3804</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Breadcrumb-mount.html">3844</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Checkbox-mount_base.html">3666</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Checkbox-mount.html">3564</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/CheckboxBase-mount_base.html">3200</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/CheckboxBase-mount.html">3109</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/ChoiceGroup-mount_base.html">6053</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/ChoiceGroup-mount.html">6117</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/ComboBox-mount_base.html">1604</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/ComboBox-mount.html">1671</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/CommandBar-mount_base.html">12639</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/CommandBar-mount.html">12489</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/ContextualMenu-mount_base.html">14580</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/ContextualMenu-mount.html">14769</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/DefaultButton-mount_base.html">1837</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/DefaultButton-mount.html">1880</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/DetailsRow-mount_base.html">4791</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/DetailsRow-mount.html">4916</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/DetailsRowFast-mount_base.html">4843</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/DetailsRowFast-mount.html">4923</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/DetailsRowNoStyles-mount_base.html">4651</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/DetailsRowNoStyles-mount.html">4790</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Dialog-mount_base.html">4027</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Dialog-mount.html">4149</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/DocumentCardTitle-mount_base.html">729</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/DocumentCardTitle-mount.html">757</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Dropdown-mount_base.html">4360</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Dropdown-mount.html">4323</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/FocusTrapZone-mount_base.html">2664</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/FocusTrapZone-mount.html">2701</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/FocusZone-mount_base.html">2586</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/FocusZone-mount.html">2585</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/GroupedList-mount_base.html">2584</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/GroupedList-mount.html">2991</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/GroupedList-virtual-rerender_base.html">1524</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/GroupedList-virtual-rerender.html">1547</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">2236</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/GroupedList-virtual-rerender-with-unmount.html">2263</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/GroupedListV2-mount_base.html">760</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/GroupedListV2-mount.html">732</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/GroupedListV2-virtual-rerender_base.html">680</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/GroupedListV2-virtual-rerender.html">688</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">749</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">728</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/IconButton-mount_base.html">2644</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/IconButton-mount.html">2613</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Label-mount_base.html">937</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Label-mount.html">967</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Layer-mount_base.html">5758</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Layer-mount.html">5705</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Link-mount_base.html">1070</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Link-mount.html">1078</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/MenuButton-mount_base.html">2267</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/MenuButton-mount.html">2232</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/MessageBar-mount_base.html">2974</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/MessageBar-mount.html">3042</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Nav-mount_base.html">4431</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Nav-mount.html">4396</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/OverflowSet-mount_base.html">1793</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/OverflowSet-mount.html">1772</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Panel-mount_base.html">3267</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Panel-mount.html">3360</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Persona-mount_base.html">1698</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Persona-mount.html">1797</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Pivot-mount_base.html">2116</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Pivot-mount.html">2186</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/PrimaryButton-mount_base.html">2037</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/PrimaryButton-mount.html">2043</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Rating-mount_base.html">9606</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Rating-mount.html">9951</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/SearchBox-mount_base.html">2079</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/SearchBox-mount.html">2148</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Shimmer-mount_base.html">4015</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Shimmer-mount.html">3807</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Slider-mount_base.html">2861</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Slider-mount.html">2838</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/SpinButton-mount_base.html">6312</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/SpinButton-mount.html">6200</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Spinner-mount_base.html">1047</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Spinner-mount.html">1037</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/SplitButton-mount_base.html">4232</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/SplitButton-mount.html">4211</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Stack-mount_base.html">1086</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Stack-mount.html">1089</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/StackWithIntrinsicChildren-mount_base.html">3270</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/StackWithIntrinsicChildren-mount.html">3331</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/StackWithTextChildren-mount_base.html">6767</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/StackWithTextChildren-mount.html">6849</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/SwatchColorPicker-mount_base.html">14018</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/SwatchColorPicker-mount.html">14099</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/TagPicker-mount_base.html">3673</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/TagPicker-mount.html">3595</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/TeachingBubble-mount_base.html">114739</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/TeachingBubble-mount.html">117125</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Text-mount_base.html">1061</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Text-mount.html">1087</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/TextField-mount_base.html">2171</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/TextField-mount.html">2217</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/ThemeProvider-mount_base.html">1975</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/ThemeProvider-mount.html">1976</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/ThemeProvider-virtual-rerender_base.html">1357</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/ThemeProvider-virtual-rerender.html">1380</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2799</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2787</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Toggle-mount_base.html">1494</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/Toggle-mount.html">1516</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/buttonNative-mount_base.html">700</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25452/perf-test/buttonNative-mount.html">707</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: packages/react-components/react-datepicker/src/index.ts [hunk 2]
```diff
@@ -1,16 +1,58 @@
 // TODO: replace with real exports
 export {};
-export { DatePicker, datePickerClassNames, renderDatePicker_unstable, useDatePickerStyles_unstable, useDatePicker_unstable } from './DatePicker';
+export {
```
Comment (spmonahan) [inline packages/react-components/react-datepicker/src/index.ts:3]: Looks like you've got some unrelated changes in this PR.

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Review by ling1726 — CHANGES_REQUESTED** [review]
Please split this PR to check in the autogenerated package boilerplate first.

Also like @spmonahan said, there are unrelated changes here.

File: README.md (hunk 1 above)
Comment (fabricteam) [inline README.md:35]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=25452&commitId=4c5ea12ee3722c08ba4afab3029338d8456a2f07&env=prod)

✅ There was 4 screenshots added, 0 screenshots removed, 1041 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by layershifter** [comment]
@tomi-msft I see that you have opened #26656, can be this closed?

**Comment by tomi-msft** [comment]
This PR was split out to #26656
